.github/ISSUE_TEMPLATE/010-specification-review.md

```diff
@@ -10,41 +10,46 @@ assignees: ''
-  - The group where the work on this specification is (or is intended to be in the future):
-
-We recommend the explainer to be in [Markdown](https://github.github.com/gfm/). On top of the [usual information expected in the explainer](https://w3ctag.github.io/explainers), it is strongly recommended to add:
-
-  - Links to major pieces of multi-stakeholder review or discussion of this specification: 
-  - Links to major unresolved issues or opposition with this specification:
-  - Links to external status/issue trackers for this specification (publicly visible, e.g. Chrome Status):
+  - Relevant time constraints or deadlines: [please provide]
+  - The group where the work on this specification is being done:
+  - Major unresolved issues or opposition with this specification:
+  - This work is being funded by: 
 
 You should also know that...
 
 [please tell us anything you think is relevant to this review]
 
-We'd prefer the TAG provide feedback as (please select one):
+We'd prefer the TAG provide feedback as (please delete all but the desired option):
+
+  🐛open issues in our GitHub repo for **each point of feedback**
+  🦋open a single issue in our GitHub repo **for the entire review**
+  💬leave review feedback as a **comment in this issue** and @-notify [github usernames]
+
+------------------------------------------------------------------------------------
+CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING
 
-  - [ ] open issues in our GitHub repo for each point of feedback
-  - [ ] open a single issue in our GitHub repo for the entire review
-  - [ ] leave review feedback as a comment in this issue and @-notify [github usernames]
+Please preview the issue and check that the links work before submitting.
 
---------------------------
+In particular, if anything links to a URL which requires authentication (e.g. Google document), please make sure anyone with the link can access the document. We would prefer fully public documents though, since we work in the open.
 
-_Please preview the issue and check that the links work before submitting._ In particular, if anything links to a URL which requires authentication (e.g. Google document), please make sure anyone with the link can access the document.
+¹ We require an explainer to give the relevant context for the spec review, even if the spec has some background information. For background, see our [explanation of how to write a good explainer](https://w3ctag.github.io/explainers). We recommend the explainer to be in [Markdown](https://github.github.com/gfm/).
 
-¹ For background, see our [explanation of how to write a good explainer](https://w3ctag.github.io/explainers).
+² A Security and Privacy questionnaire helps us understand potential Security and Privacy issues and mitigations for your design, and can save us asking redundant questions. See https://www.w3.org/TR/security-privacy-questionnaire/.
```

Comment:
Downcase the second set of "security and privacy"

Reply:
Done.